src/codewhisperer/service/securityIssueHoverProvider.ts

```diff
@@ -48,21 +48,12 @@ export class SecurityIssueHoverProvider extends SecurityIssueProvider implements
         markdownString.isTrusted = true
         markdownString.supportHtml = true
         markdownString.supportThemeIcons = true
-        markdownString.baseUri = vscode.Uri.file(
-            path.join(globals.context.extensionPath, 'dist/src/codewhisperer/images/')
-        )
+        markdownString.baseUri = vscode.Uri.file(path.join(globals.context.extensionPath, 'resources/images/'))
```

Comment:
If these are icons (I didn't know what "image" referred to in the previous PR), they likely belong somewhere in `resources/icons/`

Reply:
I was thinking that initially, but it seems like the format of these svgs are different and maybe causing issues in generating the font file.
```
Found a clipPath element in the icon "aws-codewhisperer-severity-critical" the result may be different than expected
```

If I try to use the generated icons in the hover or webview, the icon doesn't render properly. Since I'm not using it this way I figured I'd leave it outside of `resources/icons/`

Reply:
hmm. please add that explanation to the commit message.